cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/SchemaConverter.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright © 2014 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.template.etl.common;
+
+import co.cask.cdap.api.data.schema.Schema;
+import co.cask.cdap.api.data.schema.UnsupportedTypeException;
+import co.cask.cdap.internal.io.ReflectionSchemaGenerator;
+
+import java.lang.reflect.Type;
+import java.util.Iterator;
+import java.util.Map;
+
+/**
+ * Helper class for converting a {@link Schema} into a hive schema.
+ */
+public class SchemaConverter {
+  private static final ReflectionSchemaGenerator schemaGenerator = new ReflectionSchemaGenerator();
+
+  private SchemaConverter() { }
+
+  /**
+   * Create a hive schema from the given type using reflection.
+   *
+   * @param type type of class to derive the schema from.
+   * @return hive schema that can be used in a create statement.
+   * @throws UnsupportedTypeException if there is an unsupported type.
+   */
+  public static String toHiveSchema(Type type) throws UnsupportedTypeException {
```

Comment:
Remove this method since it's not used. Also remove the schemaGenerator as well.